CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 0.4.0 (2022-01-05)
```

Comment:
0.3.0 hasn't been released yet. Can we include this change in 0.3.0?

Reply:
We can do that. I will change this info in this PR and on #22 PR as well.